app/controllers/medicaid/member_steps_controller.rb

```diff
@@ -8,12 +8,12 @@ def current_member
       @_current_member ||= member_from_form || member_from_querystring
     end
 
-    private
-
     def next_path
       next_member_path || super
     end
 
+    private
```

Comment:
next_path could probably stay private if we use the inheritance structure in IntroMaritalStatusIndicateSpouseController

Reply:
yeah for sure! the reason I moved it out was so I could test it like we do for most of our controllers. What are your thoughts on those tests, are they providing value or should we keep `next_path` private?

Reply:
Got it - I was confused before and was conflating a few things. looks good